packages/web-app/.storybook/en.json

```diff
@@ -1010,8 +1012,13 @@
   "theme": "theme",
   "Theme": "Theme",
   "There is no caves related to this group currently": "There is no caves related to this group currently.",
+  "There is currently no location for this entrance": "There is currently no location for this entrance.",
```

Comment:
meme remarque ici. Il y a des "." coté droit qui devrait etre aussi coté gauche ! sinon les traducteurs des autres langues vont l'oublier !
remarque valable pour la ligne au dessus et en dessous également

Reply:
We thought that the left side is the key and the right side is the information. In that case we did not think that we need to write the full sentence in the left side, and the translation will be on the right side only. 
Our mistake, we will update this and add the '.' you ask for. 

By the way, which language do you prefer between french and english in PR comments ? Because both are used and we think that it's better if we use only one ;)

Reply:
No matter for the language. All dev on the project are currently french speaker (with english knowledge)